samples/net/coap_server/src/coap-server.c

```diff
@@ -131,6 +131,27 @@ static int well_known_core_get(struct coap_resource *resource,
 	return r;
 }
 
+static void payload_dump(const char *s, u8_t *data,
+		unsigned int len, unsigned int payloadlen)
+{
+	unsigned int i, loop_count;
+
+	loop_count = len - payloadlen;
+
+	printk("payload message =  %s\n", s);
+	for (i = loop_count; i < len; i++) {
+		printk("%02x ", data[i]);
+	}
+
+	printk("\n");
+
+	for (i = loop_count; i < len; i++) {
+		printk(" %c ", data[i]);
+	}
+	printk("\npayloadlen = %u bytes\n", payloadlen);
+}
+
+
```

Comment:
remove this line. only one blank line is required between function declarations.